packages/legacy/core/App/screens/Home.tsx

```diff
@@ -37,12 +37,13 @@ const Home: React.FC<HomeProps> = () => {
   const { ColorPallet } = useTheme()
   const [store, dispatch] = useStore()
   const { start } = useTour()
+  const [showNotifications] = useState(true)
```

Comment:
This state variable isn't changed and is used in more than one component. Would it be better adding it as a config option? (`TOKENS.CONFIG`)

Reply:
Otherwise I don't really see the purpose, as it's not overideable and isn't ever changed. If that's fine then maybe we can just remove it entirely?

Reply:
@bryce-mcmath My bad , it is a redundant state in this commit. Actually we have 2 stories to improve the Notifications list. There is a pair of chevron icons to expand/shrink the notification and toggle with the home default screen. The 'showNotifications' is the toggle control bind to the chevron onPress event, so I added it to the props for future usage. I can remove the chevron and this props from HomeHeaderView, or we just keep it in this updating, I will submit new PR for the next story this week to implement them.